src/main/java/readAndWrite/ReadAndWrite.java

```diff
@@ -0,0 +1,108 @@
+package readAndWrite;
+
+import java.io.*;
+import java.util.*;
+
+public class ReadAndWrite implements Map<Integer, MenuSingleRow> {
```

Comment:
Why do you implement it and not extend it ? And what is this class name ? Why do you even implement/extend the Map interface?